modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc

```diff
@@ -154,4 +159,85 @@ spec:
     secret: binding-request-72ddc0c540ab3a290e138726940591debf14c581 <1>
 ...
 ----
-<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
\ No newline at end of file
+<1> Unique name of the `Secret` resource generated by the {servicebinding-title}.
```

Comment:
```suggestion
<1> The unique name of the `Secret` resource that {servicebinding-title} generates.
```
- Use articles and the present tense.
- Avoid the passive voice.

Reply:
Does the qualifier, `generated by the {servicebinding-title}` help the user identify the `Secret` resource? If not, consider dropping it.

Reply:
Fixed